arraylist/src/main/java/ua/hillel/ArrayList.java

```diff
@@ -1,68 +1,125 @@
 package ua.hillel;
 
+import java.util.Objects;
+
 /**
  * Implement a list that is based on array. This resizable data structure
  * is based on an array and is a simplified version of {@link java.util.ArrayList}.
  */
 public class ArrayList<T> implements List<T> {
 
+    private Object[] elements;
+
     @Override
     public void add(T element) {
-
+        if (elements.length == size()) {
+            Object[] newElements = new Object[size() * 2];
+            System.arraycopy(element, 0, newElements, 0, size());
+            elements = newElements;
+        }
+        elements[size()] = element;
     }
 
     @Override
     public void add(int index, T element) {
-
+        if (elements.length == size()) {
+            Object[] newElements = new Object[size() * 2];
+            System.arraycopy(element, 0, newElements, 0, size());
+            elements = newElements;
+        }
+        elements[index] = element;
     }
 
     @Override
     public T get(int index) {
-        return null;
+        if (index < 0 ||index >= elements.length) {
+            throw new IndexOutOfBoundsException("Input correct index!");
+        }
+        return (T) elements[index];
     }
 
     @Override
     public T get(T element) {
-        return null;
+        if (element == null) {
+            throw new IllegalArgumentException("Input correct data!");
```

Comment:
Please make the message more clear. If element = null, then the message should be like `Parameter [element] cannot be null!`